build/migration-guide.md

```diff
@@ -0,0 +1,109 @@
+# Migration to PoS user guide
+
+These instructions are exclusively for users interested in using Nimiq’s Migration Tools to transition from PoW to PoS. If you wish to become a validator post-migration, please refer to this [documentation](becoming-a-validator.md). This is the user guide for the registration tool. For a complete understanding of the migration technicalities, refer to this [document](/learn/protocol/migration).
+
+## Validator Registration Tool
+
+The Validator Registration tool facilitates the registration process for users migrating from PoW to PoS during the pre-registration phase. Users register within the PoW chain. The Validator Registration tool operates in two distinct modes:
+
+- Address and corresponding keys generation
+- Validator registration via transactions
+
+ First, you need to clone the [core-js repository](https://github.com/nimiq/core-js?tab=readme-ov-file#quickstart) and follow the instructions.
+
+### Step 1: Generate the keys
+
+Execute the script without parameters to generate the validator address, signing key, and voting key:
+
+```shell
+node validator-registration.js
```

Comment:
After you've cloned `core-js` you have to change directory accordingly in order to be able to execute the `validator-registration.js` script.

The script can be found in `clients/nodejs/validator-registration.js` so first a
```
cd clients/nodejs
```
before you can run the script.